ember-flight-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashicorp/ember-flight-icons",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Shouldn't this bump a major because flight-icons did?

Reply:
mm, I am not sure :) I remember we discussed these cases, but I don't remember what was the decision for cases like this. in any case, better to be on the safe side, you're right. I'll push a change.

Reply:
@MelSumner pushed v `2.0.0`